xdsl/dialects/omp.py

```diff
@@ -147,6 +149,31 @@ class VariableCaptureKind(StrEnum):
     VLAType = "VLAType"
 
 
+class DataSharingClauseKind(StrEnum):
+    private = auto()
```

Comment:
I believe we have the convention that all enum fields should be in All caps (even though this would mean that the string field has to be filled in manually.